openslides/motions/models.py

```diff
@@ -703,7 +703,7 @@ class Category(RESTModelMixin, models.Model):
     """
     access_permissions = CategoryAccessPermissions()
 
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
@lostwarrior404 I think for this change in the databasestructure a migration is necessary. What happens if there are already two categories with the same name in the db?